services/tenant-ui/frontend/src/helpers/formatters.ts

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
Yeah I like this, can start moving the other table formatters in here.
Think it's intended on specifically containing just the object > datatable row formatters? Or other things too.
Could specify the file name as tableFormatters or something then maybe 🤷